its/src/org/sonarlint/eclipse/its/OpenInIdeTest.java

```diff
@@ -80,7 +84,7 @@ public class OpenInIdeTest extends AbstractSonarQubeConnectedModeTest {
     .keepBundledPlugins()
     .setEdition(Edition.DEVELOPER)
     .activateLicense()
-    .setSonarVersion(System.getProperty("sonar.runtimeVersion", "LATEST_RELEASE"))
+    .setSonarVersion(System.getProperty("sonar.runtimeVersion", "DEV"))
```

Comment:
Is it a temporary fix? Do we really want to test against SQ DEV in the long run? If not, maybe add a comment explaining why we had to use DEV.

Reply:
Missed that